test/custom/NFTDestructed.test.js

```diff
@@ -19,7 +19,8 @@ const {
 const { MARKETS } = require('../helpers/constants');
 
 describe('NFT Contract Self-Destructed', async () => {
-    MARKETS.map((marketName) => {
+    // The Nouns NFT contract cannot self-destruct
+    MARKETS.filter(m => m !== MARKET_NAMES.NOUNS).map((marketName) => {
```

Comment:
fix: need to import `MARKET_NAMES`

Reply:
(addressed this but can't push)

Reply:
https://github.com/PartyDAO/partybid/pull/43/commits/7c246cc5dcbc97d4881719fd7f87ee879cab95d6